jsonschema_gentypes/cli.py

```diff
@@ -28,14 +29,26 @@ def main() -> None:
     parser = argparse.ArgumentParser(usage="Generate the Python type files from the JSON schema files")
     parser.add_argument("--config", default="jsonschema-gentypes.yaml", help="The configuration file")
     parser.add_argument("--skip-config-errors", action="store_true", help="Skip the configuration error")
+    parser.add_argument("--json-schema", help="The JSON schema")
+    parser.add_argument("--python", help="The generated python file")
     args = parser.parse_args()
 
-    schema_data = pkgutil.get_data("jsonschema_gentypes", "schema.json")
-    assert schema_data
-    try:
-        config = validate.load(args.config, json.loads(schema_data))
-    except validate.ValidationError as error:
-        LOG.error(error)
+    if args.python is not None or args.json_schema is not None:
+        if args.python is None or args.json_schema is None:
+            print("If you specify the argument --python or --json-schema the other one is required")
+            sys.exit(1)
+        config: configuration.Configuration = {
+            "generate": [{"source": args.json_schema, "destination": args.python}]
```

Comment:
As I understand, this use case is a sort of simple mode, we only give an input schema and a python output on the command line and config argument is completely ignored. No possibility to configure headers, callbacks, root_name, etc.

Reply:
Yes, it's just tu have a quick comment that didn't need to create a new file...